CHANGELOG.md

```diff
@@ -6,9 +6,9 @@
 
 * The Mapbox Navigation SDK for iOS license has changed from the ISC License to the Mapbox Terms of Service. ([#2808](https://github.com/mapbox/mapbox-navigation-ios/pull/2808))
 * You can now install MapboxNavigation using Swift Package Manager, but you can no longer install it using Carthage. If you previously installed MapboxNavigation using Carthage, use Swift Package Manager instead. ([#2808](https://github.com/mapbox/mapbox-navigation-ios/pull/2808))
-* MapboxNavigation now depends on [MapboxMaps v10.0.0-rc.1](https://github.com/mapbox/mapbox-maps-ios/). ([#3077](https://github.com/mapbox/mapbox-navigation-ios/pull/3077))
-* MapboxNavigation now depends on MapboxNavigationNative v53.0.1. ([#3077](https://github.com/mapbox/mapbox-navigation-ios/pull/3077))
-* MapboxNavigation now depends on MapboxCommon v14.2.0. ([#3077](https://github.com/mapbox/mapbox-navigation-ios/pull/3077))
+* MapboxNavigation now depends on [MapboxMaps v10.0.0-rc.2](https://github.com/mapbox/mapbox-maps-ios/). ([#3077](https://github.com/mapbox/mapbox-navigation-ios/pull/3077))
+* MapboxNavigation now depends on MapboxNavigationNative v54.0.0. ([#3077](https://github.com/mapbox/mapbox-navigation-ios/pull/3077))
+* MapboxNavigation now depends on MapboxCommon v14.0.1. ([#3077](https://github.com/mapbox/mapbox-navigation-ios/pull/3077))
```

Comment:
Please also keep links for both #3077 and #3113, it might be useful for those who're going to use this update to understand that we've made two updates.